pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerGauge.java

```diff
@@ -42,7 +42,8 @@ public enum ServerGauge implements AbstractMetrics.Gauge {
   // Upsert metrics
   UPSERT_PRIMARY_KEYS_COUNT("upsertPrimaryKeysCount", false),
   // Dedup metrics
-  DEDUP_PRIMARY_KEYS_COUNT("dedupPrimaryKeysCount", false);
+  DEDUP_PRIMARY_KEYS_COUNT("dedupPrimaryKeysCount", false),
+  CONSUMPTION_RATE_TO_LIMIT_RATIO_PERCENT("ratio", false);
```

Comment:
This name is a bit confusing: is a rate or a ratio? can we have a more telling name?

Reply:
It's the ratio of "consumption rate" over "consumption rate limit" in terms of percentage. For example, if consumption rate is 45 and consumption rate limit is 50, the ratio_percentage will be 90%.
Let me know if you can come up with a better name.

Reply:
`CONSUMPTION_QUOTA_UTILIZATION`?